codespell_lib/data/dictionary.txt

```diff
@@ -17358,6 +17373,7 @@ inconlusive->inconclusive
 inconsisent->inconsistent
 inconsisently->inconsistently
 inconsisntency->inconsistency
+inconsistance->inconsistence
```

Comment:
Suggested word `inconsistance->inconsistence` is not a valid word in some of the major online dictionaries, hunspell, aspell, Chrome browser, etc
https://www.lexico.com/search?filter=en_dictionary&query=inconsistence
https://www.ldoceonline.com/spellcheck/english/?q=inconsistence
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=inconsistence

```text
inconsistance->inconsistency
inconsistancies->inconsistencies
```

Reply:
[6e3b2cd](https://github.com/codespell-project/codespell/pull/2414/commits/6e3b2cdc7f6be58f2df27dfd175a734355743c44)